salt/state.py

```diff
@@ -1930,6 +1930,7 @@ def call(self, low, chunks=None, running=None, retries=1):
                         # run the state call in parallel, but only if not in a prereq
                         ret = self.call_parallel(cdata, low)
                     else:
+                        self.format_slots(cdata)
```

Comment:
Is there any reason why `format_slots` isn't called for parallel states inside of `_call_parallel_target`?